spec/features/user/logged_in_user_can_logout_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'a user who is logged in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.